src/util/pg.py

```diff
@@ -629,21 +648,79 @@ def get_depending_views(cr, table, column):
     return cr.fetchall()
 
 
+class ColumnList(UserList, sql.Composable):
+    """
+    Encapsulate a list of elements that represent column names.
+    The resulting object can be rendered as string with leading/trailing comma or an alias.
+
+    Examples:
+    ```
+    >>> columns = ColumnList(["id", "field_Yx"])
```

Comment:
Wrong API. The quoted list should also be provided.
```suggestion
    >>> columns = ColumnList(["id", "field_Yx"], ["id", '"field_Yx"'])
```

Actually, the `ColumnList` objects are not created directly but through the `util.get_columns` function or via the `using` method.